tests/keras_tests/exporter_tests/keras_fake_quant/networks/tfoplambda_test.py

```diff
@@ -59,7 +58,7 @@ def run_checks(self):
         assert self.loaded_model.layers[7].output.ref() == self.loaded_model.layers[9].input.ref()
 
         assert isinstance(self.loaded_model.layers[10], TFOpLambda)
-        assert self.loaded_model.layers[10].function == tf.add
+        assert self.loaded_model.layers[10].symbol == 'math.add'
```

Comment:
Is this relevant only for 2.15 or would it work also in other versions? I don't understand why in some places you need to check .function or .symbol and in some places just .symbol